install_config/install/prerequisites.adoc

```diff
@@ -136,6 +136,19 @@ RAM.
 When planning an environment with multiple masters, a minimum of three etcd
 hosts as well as a load-balancer between the master hosts, is required.
 
+The Openshift master caches deserialized versions of resources fairly
+aggressively to ease CPU load.  However, in smaller clusters of less than 1000
+pods, this cache can waste a lot of memory for negligible CPU load reduction.
```

Comment:
Yes, but I'm not sure how to say that concisely.  "Pods" was the measurement of cluster size used in the previous paragraph so I just used that.